workspaces/variables.md

```diff
@@ -33,6 +33,19 @@ env | grep CODER_
         <td><code>CODER_USERNAME</code></td>
         <td>Your user name</td>
     </tr>
+    <tr>
+        <td><code>CODER_URL</code></td>
+        <td>The base URL of Coder</td>
+    </tr>
+    <tr>
+        <td><code>CODER_WP_NAME</code></td>
+        <td>The name of the workspace provider hosting the workspace</td>
+    </tr>
+    <tr>
+        <td><code>CODER_ASSETS_ROOT</code></td>
+        <td>The directory where coder adds coder-specific assets during
+            workspace creation, such as the coder-cli binary</td>
```

Comment:
```suggestion
            workspace creation, such as the `coder-cli` binary</td>
```

Reply:
pushed in https://github.com/cdr/docs/pull/643/commits/dc86f04babd66f9d8743a0d3f139248bfb6715bb - will need to check preview